addons/mail/static/src/components/message/message.js

```diff
@@ -116,6 +116,15 @@ export class Message extends Component {
         return this.messageView.message.date.format(getLangDatetimeFormat());
     }
 
+    /**
+     * Get the date time of the edited message at current user locale time.
+     *
+     * @returns {string}
+     */
+    get editdatetime() {
+        return this.messageView.message.editDate.format(getLangDatetimeFormat());
+    }
+
```

Comment:
No getter in component, replace this by introducing field `editDatetime` on JS model `Message`.

Reply:
done.